Themes/default/languages/Who.english.php

```diff
@@ -117,24 +141,36 @@
 $txt['whoallow_membergroups'] = 'Managing membergroups';
 $txt['whoallow_regcenter'] = 'Viewing the registration center';
 $txt['whoallow_paidsubscribe'] = 'Managing paid subscriptions';
-$txt['whoallow_moderate'] = 'Viewing the <a href="' . $scripturl . '?action=moderate">Moderation Center</a>.';
-$txt['whoallow_mlist'] = 'Viewing the <a href="' . $scripturl . '?action=mlist">memberlist</a>.';
+// argument(s): $scripturl
+$txt['whoallow_moderate'] = 'Viewing the <a href="%1$s?action=moderate">Moderation Center</a>.';
+// argument(s): $scripturl
+$txt['whoallow_mlist'] = 'Viewing the <a href="%1$s?action=mlist">memberlist</a>.';
 $txt['whoallow_optimizetables'] = 'Optimizing the database tables.';
 $txt['whoallow_repairboards'] = 'Repairing the database tables.';
-$txt['whoallow_search'] = '<a href="' . $scripturl . '?action=search">Searching</a> the forum.';
+// argument(s): $scripturl
+$txt['whoallow_search'] = '<a href="%1$s?action=search">Searching</a> the forum.';
 $txt['whoallow_search2'] = 'Viewing the results of a search.';
-$txt['whoallow_stats'] = 'Viewing the <a href="' . $scripturl . '?action=stats">forum stats</a>.';
+// argument(s): $scripturl
+$txt['whoallow_stats'] = 'Viewing the <a href="%1$s?action=stats">forum stats</a>.';
```

Comment:
none of the `whoallow_` ones are being replaced, I believe you need to so here

https://github.com/SimpleMachines/SMF2.1/blob/017e10b91144d1d68754947e6364944c9902b348/Sources/Who.php#L428